app.go

```diff
@@ -29,6 +29,10 @@ type App struct {
 	HideHelp bool
 	// Boolean to hide built-in version flag
 	HideVersion bool
+	// Display commands by category
+	CategorizedHelp bool
+	// Populate when displaying AppHelp
+	Categories map[string]Commands
```

Comment:
Just `Categories` (just since it is a calculated field which we wouldn't want to risk getting out of sync).